c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -310,7 +310,7 @@ help:
 	@echo "Utility targets:"
 	@echo
 	@echo  "- update		Update the project (Git, Node, Pip)"
-	@echo  "- update-git-submodules	Update the Git submodules"
+	@echo  "- update-git-submodules	Update the Git, submodules, and clean unrevised file"
```

Comment:
`git pull --rebase origin <branch>` but I just see that I'm wrong, then I update the pullrequest